### ISSUE TITLE
FATAL: makesedona failed

### ISSUE THREAD

**Issue by Clem-acc** [linsong/sedona#23]
Im stuck at this point, could really use some help.

Thanks,

SS

WARNING: java_home environment variable not setup correctly

options.version =  1.2.29
options.test    =  None
options.run     =  False
options.scode   =  /home/scott/sedonadev/scode/x86-test.xml
options.app     =  /home/scott/sedonadev/apps/test.sax
Compile [sedona.jar]
  Javac [/home/scott/sedonadev/src/sedona/src]
sh: 1: /usr/lib/jvm/java-7-openjdk-amd64/bin/javac: not found
**
** FAILED [/home/scott/sedonadev/lib/sedona.jar]
**
Traceback (most recent call last):
  File "/home/scott/sedonadev/adm/makedev.py", line 88, in <module>
    raise env.BuildError("FATAL: makesedona failed")   
env.BuildError: 'FATAL: makesedona failed'

**Comment by AndreySV** [comment]
Your first error is looks like related to missing openjdk. Do you have it in your system?
Second error don't contain any information about error. What test have failed?

BTW IMHO better place to ask for help is sedona mailing list: https://groups.google.com/forum/#!forum/sedonadev

**Comment by linsong** [comment]
I agree with @AndreySV , the first error is caused by missing jdk, if you use debian/ubuntu, you can install some package named 'default-jdk'. 

after the first error is resolved, pls let us know if the second error still happens.
